src/iframe-worker.js

```diff
@@ -0,0 +1,75 @@
+import { getObjectUrlForScript, fetchScriptUrlAsBlob } from './worker-utils'
+
+class IframeWorker extends EventTarget {
+  constructor(scriptUrl, { name }) {
+    super()
+
+    this.name = name
+    this.iframe = document.createElement('iframe')
+    this.iframe.sandbox = 'allow-scripts'
+
+    const source = `
+      <script>
+        const getObjectUrlForScript = ${getObjectUrlForScript.toString()}
+        const fetchScriptUrlAsBlob = ${fetchScriptUrlAsBlob.toString()}
+        const init = async () => {
```

Comment:
This chunk of code won't get transpiled. Could that be a problem?

Reply:
Hmmm, I think it doesn't do anything we'd need transpiling but maybe I could test the same approach of `toString()` for init

Reply:
Good idea! That would essentially convert to string at run time after having been transpiled during build time

Reply:
A ver, had to make sure data came in as params to keep closure: 0fbe971dbd80920b28a6a1abdb5798d46b0c9775
Let me know what you think.

Reply:
Looks good